### DISCUSSION TITLE
How to set a specific refresh time when using $this->setRefreshMethod('refresh')

### DISCUSSION THREAD

**Discussion by jdsm111200** [rappasoft/laravel-livewire-tables#1269] [Q&A]
I am trying to process certain information when the table polls, so I need to use a custom method when it refreshes. However, I am unable to set the frequency at which the table updates, and it defaults to 2 seconds, which can overload the server. I would appreciate your help if it's possible to use setRefreshMethod and set a specific refresh time. I have tried reading various forums and the documentation, but I haven't found any answers.

**Comment by lrljoe** [comment]
So you need to multiply the seconds by 1000, but your configure() method in your Table Component should contain something like the below:

```
public function configure(): void
{
    $this->setRefreshTime(5000); // Component refreshes every 5 seconds
}
```

What exactly are you trying to process, as there may (will probably) be a much easier/more efficient solution, and there are a variety of options available to push/pull data between different components etc.
